changelog/1.25.0.md

```diff
@@ -4,8 +4,9 @@ description: "Released on 11/17/2021"
 ---
 
 > The final patch release of Kubernetes 1.19 was published on 28 October 2021.
-> As such, Coder v1.26 (and later) will require the use of Kubernetes 1.20 or
-> later. See Coder's [version support policy] for more information.
+> As such, the _subsequent_ versions of Coder (v1.26 later) will require the use
```

Comment:
@khorne3 looks like we're missing a word, I'd expect "1.26 or later" or "1.26 and later" but otherwise looks good to me!